kong-3.1.0-0.rockspec

```diff
@@ -42,6 +42,8 @@ dependencies = {
   "lua-resty-acme == 0.8.1",
   "lua-resty-session == 3.10",
   "lua-resty-timer-ng == 0.2.0",
+  "expadom >= 0.1.0",
```

Comment:
since expadom is in the "ustable range" (semver, before 1.0) this one should probably be pinned exactly to this version, instead of open ended.